packages/build/src/plugins_core/blobs_upload/index.ts

```diff
@@ -72,7 +72,8 @@ const coreStep: CoreStepFunction = async function ({
         systemLog(`Uploading blob ${key}`)
 
         const { data, metadata } = await getFileWithMetadata(key, contentPath, metadataPath)
-        await blobStore.set(key, data, { metadata })
+        const arrayBuffer = data.buffer.slice(data.byteOffset, data.byteOffset + data.length)
+        await blobStore.set(key, arrayBuffer, { metadata })
```

Comment:
https://github.com/netlify/build/pull/5921/files

Does this approach not work? If so, it seems simpler.

Reply:
I believe it was having issues on Node 14 still